resource-watch/resource-watch
# Back office: Explore form

![image](https://user-images.githubusercontent.com/545342/76831531-c9fc7480-6827-11ea-8a16-ad4af4d2dab8.png)

## Overview
This PR adds a new Explore form to the back office.

## Testing instructions
Go to http://localhost:9000/admin/data/explore and verify that the different sections work properly.

⚠️ **NOTE:** We're still waiting for the API to implement the `isHighlighted` property for datasets together with the `config` service. Thus, this form isn't yet _usable_ from the data perspective
